styles.css

```diff
@@ -1,50 +1,89 @@
 
 
-
-
 body {
```

Comment:
As far as I know, usually you'd avoid to put styles directly in the body, unless they are the default (fallback) styles. You don't need to change it for this exercise, but it would be useful for following exercises that you start thinking about re-usability of the styles. If you apply the styles directly to the `body`, `nav`, or any other html elements, these styles would apply _on your whole application_. 

So, imagine if this page was part of a bigger web app and you have a 'magical fashion tips' page. And you probably would want big pictures with the latest pointy hats, with centered content, instead of  a sidebar of 1fr and the main content to be only on the other 4fr. 

The best approach is to put most style attributes in custom classes, and then apply the classes to the elements that use them. This way we could have a class `sidebar-left-display` and a class `visual-articles-display` and we could apply those to either the body or a div inside the body on our different pages, so it's easily reusable.

Reply:
Noted!